### PR TITLE
Adding -c / --copy option to copy the password to the clipboard

### PR DESCRIPTION
**Pull request by MarcinZukowski**
Hope you find it useful :)

### CODE REVIEW

File: onepassword/cli.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import argparse
 import getpass
 import os
+import pyperclip
```
Comment (AdamIsrael) [inline onepassword/cli.py:4]: This needs to be added to the install_requires in setup.py

File: onepassword/cli.py [hunk 2]
```diff
@@ -34,13 +35,19 @@ def run(self):
...
+            self.produce(item)
         else:
             self.stderr.write("1pass: Could not find an item named '%s'\n" % (
                 self.arguments.item,
             ))
             sys.exit(os.EX_DATAERR)
 
+    def produce(self, item):
+        if self.arguments.copy:
+            pyperclip.copy(item.password)
```
Comment (AdamIsrael) [inline onepassword/cli.py]: I'm on the fence about whether we should notify the user that the password was copied to the clipboard. In theory, this should always either copy or print the password, or print that the item couldn't be found. No change necessary here, but I wanted to capture the idea for posterity.

File: .gitignore [hunk 3]
```diff
@@ -3,3 +3,4 @@ ENV
 dist
 1pass.egg-info
 build
+.idea
```
Comment (AdamIsrael) [inline .gitignore]: Where'd .idea come from?

**Review by AdamIsrael — CHANGES_REQUESTED** [review]
Pretty minor fix requested, plus some clarification needed.

File: .gitignore (hunk 3 above)
Comment (MarcinZukowski) [reply .gitignore]: Sorry, that's for PyCharm, I guess I should add it to my global gitignore

File: onepassword/cli.py (hunk 1 above)
Comment (MarcinZukowski) [reply onepassword/cli.py:4]: Right! Sorry :)

File: onepassword/cli.py (hunk 2 above)
Comment (MarcinZukowski) [reply onepassword/cli.py]: I've added it already with a bunch more functionality I needed. Will send that pr soon.

**Comment by MarcinZukowski** [comment]
Sorry, was busy with other stuff, will close this one and send another one.

**Pull request closed without merging**